src/clib/ctreactor.cpp

```diff
@@ -547,7 +498,7 @@ extern "C" {
     double wall_area(int i)
     {
         try {
-            return WallCabinet::at(i)->area();
+            return ConnectorCabinet::as<Wall>(i)->area();
         } catch (...) {
             return handleAllExceptions(DERR, DERR);
```

Comment:
Especially as we move toward the generated `clib`, I'm not sure it's worth jamming more disparate types into the same `Cabinet`.

Reply:
The generated CLib uses exactly one header file for all class specializations. My plan for `zeroD` is to establish two node types - `ConnectorNode` and `ReactorBase` (potentially renamed `ReactorNode` for consistency). As `ReactorNet` is another required class, the CLib interface would be implemented using three headers.